### PR TITLE
Fix and move zziplib to archiver directory

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/archiver/zziplib/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -88,6 +88,9 @@ link path=usr/lib/$(MACH64)/libzzipmmapped-0.so.13 \
     target=libzzipmmapped-0.so.13.0.62
 file path=usr/lib/$(MACH64)/libzzipmmapped-0.so.13.0.62
 link path=usr/lib/$(MACH64)/libzzipmmapped.so target=libzzipmmapped-0.so.13.0.62
+link path=usr/lib/$(MACH64)/libzzipwrap-0.so.10 target=libzzipwrap-0.so.13
```
Comment (xen0l) [inline components/archiver/zziplib/manifests/sample-manifest.p5m:91]: Why these changes?

**Review by xen0l — CHANGES_REQUESTED** [review]

File: components/archiver/zziplib/manifests/sample-manifest.p5m (hunk 1 above)
Comment (alarcher) [reply components/archiver/zziplib/manifests/sample-manifest.p5m:91]: It seems they were missing.
